### PR TITLE
Add links to erlang resources for italian section

### PR DESCRIPTION
**Pull request by neslinesli93**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s)

### Description 
A couple of links to erlang resources, written in italian

### Why is this valuable (or not)?
Because even if these links are not full books, they may still have value for native speaking (italian) people who are approaching erlang/otp programming for the first time

### For book lists, is it a book? For course lists, is it a course? etc.
Two links to sort of erlang "baby-steps"

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-it.md [hunk 1]
```diff
@@ -90,6 +91,12 @@
 * [Corso Arduino 2014](https://www.youtube.com/watch?v=WuqS4C4u42E&list=PLA27EZBY5vePO9T6YP3rH8LTTdylz69VE) (POuLpolimi Politecnico di Milano)
 
 
+### Erlang
+
+* [Erlang/OTP](http://www.khaldi.it/art/erlang/erl01/E01.html) - (Jilani KHALDI)
+* [Iniziare con Erlang Language](https://riptutorial.com/it/erlang) - [(riptutorial.com)](https://riptutorial.com)
```
Comment (eshellman) [inline free-courses-it.md:97]: no links for platforms

File: free-courses-it.md [hunk 2]
```diff
@@ -90,6 +91,12 @@
 * [Corso Arduino 2014](https://www.youtube.com/watch?v=WuqS4C4u42E&list=PLA27EZBY5vePO9T6YP3rH8LTTdylz69VE) (POuLpolimi Politecnico di Milano)
 
 
+### Erlang
+
+* [Erlang/OTP](http://www.khaldi.it/art/erlang/erl01/E01.html) - (Jilani KHALDI)
```
Comment (eshellman) [inline free-courses-it.md:96]: no need to put author in parens

**Review by eshellman — CHANGES_REQUESTED** [review]

File: free-courses-it.md (hunk 1 above)
Comment (neslinesli93) [reply free-courses-it.md:97]: Do I have to remove the entry alltogether or just the link at the end?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**